src/ui/server/controllers/insurance/policy/broker-addresses/index.ts

```diff
@@ -154,6 +155,16 @@ export const post = async (req: Request, res: Response) => {
       });
     }
 
+    const answer = payload[FIELD_ID];
+
+    const { __typename, ...chosenAddress } = addresses[answer];
+
+    const saveResponse = await mapAndSave.broker(chosenAddress, application);
+
+    if (!saveResponse) {
+      return res.redirect(PROBLEM_WITH_SERVICE);
```

Comment:
console.error?

Reply:
We have this throughout. I don't think we need a console.error here - the flow is:

`POST controller` => `mapAndSave.broker` => `saveData` => API GQL handler

In `saveData`, an error will be thrown if the API call fails.

If ths API call fails, there will also be console errors and/or an error thrown.

So they'll be plenty of logs/errors if the data saving fails, but I don't see why not, to add a specific log.

Can look into this separately and apply throughout - have created EMS-4021 to do this.